app/controllers/users.js

```diff
@@ -41,3 +42,24 @@ exports.getUsers = async (req, res, next) => {
     next(error);
   }
 };
+exports.adminLog = async (req, res, next) => {
+  try {
+    const userObject = req.body;
+    const userFound = await userService.findUser(userObject);
+    if (userFound === null) {
+      const password = helper.passEncryptred(userObject.password);
+      const userCreated = varParse.nameParse(userObject, password);
+      const admin = await helper.adminRole(userCreated);
+      await userService.createUser(admin);
+      logger.info('Admin created: ', admin.firstName);
+      res.status(201).send(admin);
+    } else {
+      const admin = await parser.adminRole(userFound);
```

Comment:
utiliza la función que ya existe:

```suggestion
      const admin = await varParse.nameParse(userFound, userFound.password, true);
```